src/main/java/com/facebook/openwifirrm/modules/ModelerUtils.java

```diff
@@ -202,4 +212,108 @@ public static double calculateTPCMetrics(
 			return Double.POSITIVE_INFINITY;
 		}
 	}
+
+	/**
+	 * Determines if two wifiscan entries should be aggregated without consideration
+	 * for any kind of obsoletion period. (Untested) should handle APs that support
```

Comment:
move "(Untested)" out of the javadoc and into an inline "TODO" comment, or open an internal task to track this